lib/mix/tasks/coh.context.ex

```diff
@@ -0,0 +1,187 @@
+defmodule Mix.Tasks.Coh.Context do
+  @shortdoc "Generates a context with functions around an Ecto schema representing a User"
+
+  @moduledoc """
+  Generates a context with functions around an Ecto schema representing a User.
+      mix coh.context Accounts name:string email:string
+  """
```

Comment:
Change this section to `## Examples`

Add an example to specify the schema name.
`coh.gen.schema Accounts name:string email:string --schema=Account`